"unknown flag: --max-args" when syncing the cluster

When I cluster-sync, I get the following error using the latest checkout:

```
devel: digest: sha256:4f7f516e9bc738f1f36d481d518c21d0a27dab066470ce663ddec73eef92bd0a size: 948
The push refers to a repository [localhost:32790/kubevirt/winrmcli]
5bcb727bed62: Layer already exists 
29272e6d7a60: Layer already exists 
dac761b2dc84: Layer already exists 
39bae602f753: Layer already exists 
devel: digest: sha256:cb105c04526e0da54443902eb18da0c15b8d6fb0dfc04df2d357c32dc012f833 size: 1165
make[1]: Leaving directory '/home/ihar/kubevirt'
unknown flag: --max-args
make: *** [Makefile:65: cluster-build] Error 1
```

It looks like --max-args is intercepted by 'ssh' and not passed into the node to call xargs.

--- a/cluster/ephemeral-provider-common.sh
+++ b/cluster/ephemeral-provider-common.sh
@@ -59,8 +59,8 @@ function build() {
         container_alias="${container_alias} ${manifest_docker_prefix}/${name}:${docker_tag} kubevirt/${name}:${docker_tag}"
     done
     for i in $(seq 1 ${KUBEVIRT_NUM_NODES}); do
-        ${_cli} ssh --prefix $provider_prefix "node$(printf "%02d" ${i})" "echo \"${container}\" | xargs --max-args=1 sudo docker pull"
-        ${_cli} ssh --prefix $provider_prefix "node$(printf "%02d" ${i})" "echo \"${container_alias}\" | xargs --max-args=2 sudo docker tag"
+        ${_cli} ssh --prefix $provider_prefix "node$(printf "%02d" ${i})" "echo \"${container}\" | xargs \-\-max-args=1 sudo docker pull"
+        ${_cli} ssh --prefix $provider_prefix "node$(printf "%02d" ${i})" "echo \"${container_alias}\" | xargs \-\-max-args=2 sudo docker tag"
     done
 }
 

